vignettes/hubverse_ms_vignette.Rmd

```diff
@@ -0,0 +1,331 @@
+Starting in the 2023-2024 season, FluSight has used the hubverse ecosystem to configure and maintain submissions of short-term forecasts (https://github.com/cdcepi/FluSight-forecast-hub/releases/tag/v1.0.0).
+
+We refer readers to the main manuscript or [the hubverse website](https://hubverse.io/) to learn more about the key concepts and definitions of terms that underlie this demonstration.
+
+# Required software
+
+The following demonstration shows how to access and analyze data from a real hub using `r R.version$string` and several of the hubverse R packages[@the_consortium_of_infectious_disease_modeling_hubs_hubverse_2025].
+
+The hubverse packages `hubData`, `hubVis`, `hubEnsembles` and `hubEvals` are best installed through the R-universe package repository to ensure you are working with the latest released versions. Note that these packages have some dependencies that are needed for the operations in this vignette, so specifying `dependencies = TRUE` is recommended. The packages can be installed from the R console using the following command:
+
+```{r install, eval=FALSE}
+install.packages(
+  pkgs = c("hubData", "hubVis", "hubEnsembles", "hubEvals"),
+  repos = c(
+    "https://hubverse-org.r-universe.dev",
+    "https://cloud.r-project.org"
+  ),
+  dependencies = TRUE
+)
+```
+
+Supporting packages `ggplot2` and `dplyr` can be installed from CRAN.
+
+```{r install-CRAN, eval=FALSE}
+install.packages(
+  pkgs = c("dplyr", "ggplot2")
+)
+```
+
+Once the required packages are installed, load the supporting packages into your R session. (Session information on all the packages used including package versions are included at the end of this vignette.)
+
+```{r libraries, message=FALSE}
+# Data munging
+library(dplyr)
+# Plotting
+library(ggplot2)
+```
+
+
+# Load forecast data
+
+CDC's FluSight Forecast Hub exists as a [GitHub repository](https://github.com/cdcepi/FluSight-forecast-hub)[@noauthor_cdcepiflusight-forecast-hub_2025]. Using hubverse tools, the data for this hub has been validated and mirrored to the cloud, so a copy of the model output for this hub hub lives at a publicly-accessible AWS S3 bucket. This means that instead of needing to download a local copy of the repository, the data for this project can be directly accessed from the cloud using `hubData` tools.
```

Comment:
```suggestion
CDC's FluSight Forecast Hub exists as a [GitHub repository](https://github.com/cdcepi/FluSight-forecast-hub)[@noauthor_cdcepiflusight-forecast-hub_2025]. Using hubverse tools, the data for this hub has been validated and mirrored to the cloud, so a copy of the model output for this hub lives at a publicly-accessible AWS S3 bucket. This means that instead of needing to download a local copy of the repository, the data for this project can be directly accessed from the cloud using `hubData` tools. The data is organised in a format we can open as an arrow dataset through R package `arrow` directly .
```